frontend/src/constants/SideNavConfig.ts

```diff
@@ -0,0 +1,23 @@
+import type { SideNavItemType } from "@/types/SideNavTypes";
+import UserMultiple from "@carbon/icons-vue/es/user--multiple/16";
+import IbmLpa from "@carbon/icons-vue/es/ibm--lpa/16";
+
+export const sideNavItems: SideNavItemType[] = [
+    {
+        name: "Manage permissions",
+        link: "/manage-permissions",
```

Comment:
good question, i should be doing what you're suggesting, I let that slipped out of my mind, in fact I should be using route name instead of links to avoid joining paths. Let me make some changes

Reply:
done here! https://github.com/bcgov/nr-forests-access-management/pull/1633/commits/dee1d22eebb62f0101d98219eb28b0f5c53a57e1